src/app/frontend/logs/component.js

```diff
@@ -278,6 +278,15 @@ export class LogsController {
   }
 
   /**
+   * Tells whether the previous logs (as opposed to the current logs) are currently showing
+   * @export
+   * @return {boolean}
+   */
+  arePreviousLogsShowing() {
+    return this.logsService.getPrevious();
```

Comment:
Use `this.logsService.getPrevious()` directly in the HTML and remove this method to be consistent with the rest of the file.

Reply:
@maciaszczykm: done